### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by RhnSaxena**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # FBI Application Guide
 
-This is a little guide that I put together to make it easier to see the requirements to become and FBI Special Agent. **This guide is still in progress, so use it at your own risk.**
+This is a little guide that I have put together to make it easier to see the requirements to become an FBI Special Agent. **This guide is still in progress, so use it at your own risk.**
 
 ![Application Process](https://www.fbijobs.gov/sites/default/files/FBI_Graphic_ApplicationProcess_SpecialAgent_Website_943x551-min.png)
-The guide is sorted into a folder for each step of the application process. Refer to the image above for more information, or to the folder structure below.
+The guide is distributed into one folder for each step of the application process. Refer to the image above for more information, or to the folder structure below.
```
Comment (willtheorangeguy) [inline README.md]: Could you please change distributed back to sorted! 

Thank you

**Comment by willtheorangeguy** [comment]
@RhnSaxena if you would make the changes above then I will merge it right away!

File: README.md (hunk 1 above)
Comment (RhnSaxena) [reply README.md]: I have changed it to 'sorted'.

Thanks.

Comment (willtheorangeguy) [reply README.md]: Thanks!

**Comment by willtheorangeguy** [comment]
Ok thank you!

**Comment by willtheorangeguy** [comment]
Merged!

**Pull request merged**